### PR TITLE
Add changelog entry for 3.6, merge stable into master

### PR DESCRIPTION
**Pull request by munkm**
This is in preparation for the 3.6 release.

### CODE REVIEW

**Comment by munkm** [comment]
@brittonsmith @matthewturk or anybody else who might know -- It looks like previously we bumped the version number to a non-dev version as a commit on the "stable" branch, and then tagged it there. Is there a reason why we can't bump the version number in this PR to a non-dev version and then once master is merged into stable we tag that version bump commit? Then our tags will be in the master branch history.

**Comment by matthewturk** [comment]
I think doing it that way would be fine! I think it's related to the issue @migueldvb reported about tags on the branches, too.

I'm good with this!

File: doc/source/reference/changelog.rst [hunk 1]
```diff
@@ -12,6 +12,353 @@ The `CREDITS file <https://github.com/yt-project/yt/blob/master/CREDITS>`_
...
+    - update energy field to be `energy_density` and have units of code 
+      pressure  `PR 2376 <https://github.com/yt-project/yt/pull/2376>`__.
+- Support for the AdaptaHOP halo finder code 
+  `PR 2385 <https://github.com/yt-project/yt/pull/2385>`__.
+- yt now supports geographic transforms and projections of data with 
+  cartopy with support from `PR 1966 <https://github.com/yt-project/yt/pull/1966>`__.
+- annotations used to work for only a single point, they now work for multiple points 
+  on a plot, see `PR 2122 <https://github.com/yt-project/yt/pull/2122>`__.
+- cosmology calculations now have support for the relativistic energy density of the 
+  unvierse, see `PR 1714 <https://github.com/yt-project/yt/pull/1714>`__.
```
Comment (Xarthisius) [inline doc/source/reference/changelog.rst:65]:
```suggestion
  universe, see `PR 1714 <https://github.com/yt-project/yt/pull/1714>`__. 
```

File: doc/source/reference/changelog.rst [hunk 2]
```diff
@@ -12,6 +12,353 @@ The `CREDITS file <https://github.com/yt-project/yt/blob/master/CREDITS>`_
...
+  the ``rotate`` kwarg in the ``arrow`` function. 
+  See `PR 2151 <https://github.com/yt-project/yt/pull/2151>`__.
+- allow for dynamic load balancing with parallel loading of timeseries 
+  data using the ``dynamic`` kwarg. `PR 2149 <https://github.com/yt-project/yt/pull/2149>`__.
+- show/hide colorbar and show/hide axes are now available for 
+  ``ProfilePlot`` s. These functions were also moved from the PlotWindow to the 
+  PlotContainer class. `PR 2169 <https://github.com/yt-project/yt/pull/2169>`__.
+- The angular momentum has been reversed compared to previous versions of yt. 
+  This may cause breaking changes in your code. 
+  See `PR 2043 <https://github.com/yt-project/yt/pull/2043>`__.
```
Comment (Xarthisius) [inline doc/source/reference/changelog.rst:77]: This has a potentially major impact on users. Can we promote it some how? Make a separate category of breaking changes or something like that?

File: doc/source/reference/changelog.rst [hunk 3]
```diff
@@ -12,6 +12,353 @@ The `CREDITS file <https://github.com/yt-project/yt/blob/master/CREDITS>`_
...
+  python versions `PR 2452 <https://github.com/yt-project/yt/pull/2452>`__.
+- update custom CSS on docs to sphinx >=1.6.1. See 
+  `PR 2199 <https://github.com/yt-project/yt/pull/2199>`__.
+- enhancing the contribution documentation on git, see 
+  `PR 2420 <https://github.com/yt-project/yt/pull/2420>`__. 
+- update documentation to correctly reference issues suitable for new 
+  contributors `PR 2346 <https://github.com/yt-project/yt/pull/2346>`__.
+- fix URLs and spelling errors in a number of the cookbook notebooks 
+  `PR 2341 <https://github.com/yt-project/yt/pull/2341>`__.
+- udpate release docs to include information about building binaries, tagging,
```
Comment (Xarthisius) [inline doc/source/reference/changelog.rst:142]:
```suggestion
- update release docs to include information about building binaries, tagging, 
```

File: doc/source/reference/changelog.rst [hunk 4]
```diff
@@ -12,6 +12,353 @@ The `CREDITS file <https://github.com/yt-project/yt/blob/master/CREDITS>`_
...
+  share the same unit registry `PR 2230 <https://github.com/yt-project/yt/pull/2230>`__.
+- fix kh2d test name  `PR 2342 <https://github.com/yt-project/yt/pull/2342>`__.
+- disable OSNI projection answer test to remove cartopy errors `PR 2350 <https://github.com/yt-project/yt/pull/2350>`__.
+
+CI related support
+""""""""""""""""""
+
+- disable coverage on OSX to speed up travis testing and avoid 
+  timeouts `PR 2076 <https://github.com/yt-project/yt/pull/2076>`__.
+- pdate travis base images on Linux and
```
Comment (Xarthisius) [inline doc/source/reference/changelog.rst:333]:
```suggestion
- update travis base images on Linux and 
```

File: doc/source/reference/changelog.rst [hunk 5]
```diff
@@ -12,6 +12,353 @@ The `CREDITS file <https://github.com/yt-project/yt/blob/master/CREDITS>`_
 contains the most up-to-date list of everyone who has contributed to the yt 
 source code.
 
+Version 3.6.0
+-------------
+
+Version 3.6.0 our next major release since 3.5.1, which was in February 
+2019. It includes roughly 180 pull requests contibuted from 39 contributors,
```
Comment (Xarthisius) [inline doc/source/reference/changelog.rst:19]:
```suggestion
2019. It includes roughly 180 pull requests contributed from 39 contributors, 
```

File: doc/source/reference/changelog.rst [hunk 6]
```diff
@@ -12,6 +12,353 @@ The `CREDITS file <https://github.com/yt-project/yt/blob/master/CREDITS>`_
...
 source code.
 
+Version 3.6.0
+-------------
+
+Version 3.6.0 our next major release since 3.5.1, which was in February 
+2019. It includes roughly 180 pull requests contibuted from 39 contributors, 
+22 of which committed for their first time to the project. 
+
+We have also updated our project govenance and contribution guidelines, which
```
Comment (Xarthisius) [inline doc/source/reference/changelog.rst:22]:
```suggestion
We have also updated our project governance and contribution guidelines, which
```

File: doc/source/reference/changelog.rst [hunk 7]
```diff
@@ -12,6 +12,353 @@ The `CREDITS file <https://github.com/yt-project/yt/blob/master/CREDITS>`_
...
+you can `view here <https://yt-project.github.io/governance/>`_ .
+
+We'd like to thank all of the individuals who contributed to this relase. There
```
Comment (Xarthisius) [inline doc/source/reference/changelog.rst:25]:
```suggestion
We'd like to thank all of the individuals who contributed to this release. There
```

File: doc/source/reference/changelog.rst [hunk 8]
```diff
@@ -12,6 +12,353 @@ The `CREDITS file <https://github.com/yt-project/yt/blob/master/CREDITS>`_
...
+are lots of new features and we're excited to share them with the community. 
+
+Major Changes and New Features
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- New frontend support for the code AMRVAC. Many thanks to Clement Robert 
+  and Niels Claes who were major contributors to this initiative. Relevant PRs include:
+    - Inital PR to support AMRVAC native data files
```
Comment (Xarthisius) [inline doc/source/reference/changelog.rst:33]:
```suggestion
    - Initial PR to support AMRVAC native data files  
```

File: doc/source/reference/changelog.rst [hunk 9]
```diff
@@ -12,6 +12,353 @@ The `CREDITS file <https://github.com/yt-project/yt/blob/master/CREDITS>`_
...
+      `PR 2384 <https://github.com/yt-project/yt/pull/2384>`__, 
+      `PR 2380 <https://github.com/yt-project/yt/pull/2380>`__, 
+      `PR 2382 <https://github.com/yt-project/yt/pull/2382>`__.
+    - testing-related PRs for AMRVAC: 
+      `PR 2379 <https://github.com/yt-project/yt/pull/2379>`__, 
+      `PR 2360 <https://github.com/yt-project/yt/pull/2360>`__.
+    - add verbosity to logging of geometry or `geometry_override` 
+      `PR 2421 <https://github.com/yt-project/yt/pull/2421>`__.
+    - add attribute to `_code_unit_attributes` specific to AMRVAC to ensure 
+      consistent renormalisation of amrvac datasets. See
```
Comment (Xarthisius) [inline doc/source/reference/changelog.rst:49]:
```suggestion
      consistent renormalisation of AMRVAC datasets. See 
```

File: doc/source/reference/changelog.rst [hunk 10]
```diff
@@ -12,6 +12,353 @@ The `CREDITS file <https://github.com/yt-project/yt/blob/master/CREDITS>`_
...
+      `PR 2379 <https://github.com/yt-project/yt/pull/2379>`__, 
+      `PR 2360 <https://github.com/yt-project/yt/pull/2360>`__.
+    - add verbosity to logging of geometry or `geometry_override` 
+      `PR 2421 <https://github.com/yt-project/yt/pull/2421>`__.
+    - add attribute to `_code_unit_attributes` specific to AMRVAC to ensure 
+      consistent renormalisation of amrvac datasets. See 
+      `PR 2357 <https://github.com/yt-project/yt/pull/2357>`__.
+    - parse AMRVAC's parfiles if user-provided  
+      `PR 2369 <https://github.com/yt-project/yt/pull/2369>`__.
+    - enssure that min_level reflects dataset that has refinement
```
Comment (Xarthisius) [inline doc/source/reference/changelog.rst:53]:
```suggestion
    - ensure that min_level reflects dataset that has refinement 
```

File: doc/source/reference/changelog.rst [hunk 11]
```diff
@@ -12,6 +12,353 @@ The `CREDITS file <https://github.com/yt-project/yt/blob/master/CREDITS>`_
...
+- extend support of the ``marker``, ``text``, ``line`` and ``sphere`` annotation 
+  callbacks to polar geometries  `PR 2466 <https://github.com/yt-project/yt/pull/2466>`__.
+- Support MHD in the GAMER frontend  `PR 2306 <https://github.com/yt-project/yt/pull/2306>`__.
+- Export data container and profile fields to AstroPy QTables and 
+  pandas DataFrames  `PR 2418 <https://github.com/yt-project/yt/pull/2418>`__.
+- Add turbo colormap, a colorblind safe version of jet.  See 
+  `PR 2339 <https://github.com/yt-project/yt/pull/2339>`__.
+- Enable exporting regular grids (i.e., covering grids, arbitrary grids and 
+  smoothed grids) to ``xarray`` `PR 2294 <https://github.com/yt-project/yt/pull/2294>`__.
+- add automatic loading of ``namelist.txt``, which contains the paramater file
```
Comment (Xarthisius) [inline doc/source/reference/changelog.rst:109]:
```suggestion
- add automatic loading of ``namelist.txt``, which contains the parameter file 
```

**Comment by Xarthisius** [comment]
LGTM!

File: doc/source/reference/changelog.rst (hunk 2 above)
Comment (munkm) [reply doc/source/reference/changelog.rst]: Ok i've moved it up to a section right after the intro to the release. Does that work?

**Pull request merged**